opengrok-tools/setup.py

```diff
@@ -6,6 +6,28 @@
 SCRIPT_DIR = os.path.dirname(os.path.realpath(__file__))
 
 
+def extract_version_from_path(*file_paths):
+    """
+    Look for a version in a file
+    see https://packaging.python.org/guides/single-sourcing-package-version/
+    :param file_paths: path to the file
+    :return: the version
+    :raises: an RuntimeError when version is not available
+    """
+
+    def read(*parts):
```

Comment:
use more descriptive name of the function

Reply:
deleted the whole function